docs/sources/datasources/cloudmonitoring.md

```diff
@@ -222,6 +222,27 @@ The Alias By field allows you to control the format of the legend keys for SLO q
 
 SLO queries use the same [alignment period functionality as metric queries]({{< relref "#metric-queries" >}}).
 
+### MQL (Monitoring Query Language) queries
+
+> **Note:** Only available in Grafana v7.4+.
+
+The MQL query builder in the Google Cloud Monitoring data source allows you to display MQL results in time series format. To get an understanding of the basic concepts in MQL, refer to [Google Cloud Monitoring documentation](https://cloud.google.com/monitoring/mql).
```

Comment:
Please change that link text as follows so that it is WYSIWYG:

[Introduction to Monitoring Query Language](https://cloud.google.com/monitoring/mql)

Reply:
fixed! thanks for feedback @osg-grafana